README.md

```diff
@@ -1,4 +1,6 @@
+<!-- markdownlint-disable-next-line-->
```

Comment:
oof, this is a bit icky - why is it needed on the first line?

Reply:
While that's an improvement, let's just disable that entire rule wholesale - inline HTML is strictly necessary for a ton of things that are totally reasonable to have in markdown.

Reply:
Disabled.